src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -269,7 +269,7 @@ public sealed partial class Builder : System.Collections.Generic.ICollection<T>,
             public int LastIndexOf(T item, int startIndex, int count, System.Collections.Generic.IEqualityComparer<T>? equalityComparer) { throw null; }
             public System.Collections.Immutable.ImmutableArray<T> MoveToImmutable() { throw null; }
             public bool Remove(T element) { throw null; }
-            public void Remove(T item, IEqualityComparer<T>? equalityComparer) { throw null; }
+            public void Remove(T element, IEqualityComparer<T>? equalityComparer) { throw null; }
```

Comment:
I believe this also needs to be updated in the implementation, otherwise the api compat check will fail.

Reply:
@eiriktsarpalis  thanks, I've updated it too.